specificatie/openapi.yaml

```diff
@@ -63,7 +63,29 @@ paths:
             type: integer
             minimum: 1
             maximum: 99999
-        - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/parameters/fields"
+        - in: query
+          name: huisnummertoevoeging
+          description: Huisnummertoevoeging behorende bij een nummeraanduiding.
+          required: false
+          schema:
+            type: string
+            pattern: "^[0-9a-zA-Z]$"
+            maxLength: 4
+            minLength: 1
+            example: "bis"
+        - in: query
+          name: huisletter
+          description: Huisletter behorende bij een nummeraanduiding.
```

Comment:
In lijn brengen met beschrijving bij postcode en huisnummer?

@CathyDingemanse Zoiets als "Zoek WOZ-objecten met huisletter in combinatie met huisnummer"?

Reply:
@CathyDingemanse @fsamwel Willen jullie de descriptions bij postcode, huisnummer, huisletter en huisnummertoevoeging checken ?